packages/diracx-web/test/e2e/jobMonitor.cy.ts

```diff
@@ -149,7 +149,7 @@ describe("Job Monitor", () => {
     cy.get(".MuiChip-label").should("be.visible");
 
     cy.get("button").contains("Apply").click();
-    cy.wait(1000);
+    cy.wait(500);
```

Comment:
Why did you decide to decrease this value?

Reply:
I decided to decrease this value because a 1-second delay seemed excessive. After testing, I found that a 500ms delay still works, making it more efficient.